tests/unittests/CoreGraphics.drawing/ImageComparison.h

```diff
@@ -39,7 +41,107 @@ class ImageComparator {
     virtual ImageDelta CompareImages(CGImageRef left, CGImageRef right) = 0;
 };
 
+struct bgraPixel {
+    uint8_t b, g, r, a;
```

Comment:
This is going to make merging exceptionally painful :(

You can force template instantiations in the .cpp file. It'll have the benefit of cleaning up the diffs so we can actually see what changed here.